.github/workflows/delete-review-app.yml

```diff
@@ -0,0 +1,87 @@
+name: Delete Review App
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+jobs:
+  delete-review-app:
+    name: Delete Review App ${{ github.event.pull_request.number }}
+    concurrency: deploy_review_${{ github.event.pull_request.number }}
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'deploy')
+    runs-on: ubuntu-latest
+    environment: review
+    steps:
+      - name: Extract configuration from tfvars
+        id: config
+        run: |
+          KEY_VAULT_NAME=$(jq -r '.key_vault_name' $TFVARS)
+          PAAS_SPACE=$(jq -r '.paas_space' $TFVARS)
+
+          if [ -z "$KEY_VAULT_NAME" ]; then
+            echo "::error ::Failed to extract key_vault_name from $TFVARS"
+            exit 1
+          fi
+
+          if [ -z "$PAAS_SPACE" ]; then
+            echo "::error ::Failed to extract paas_space from $TFVARS"
+            exit 1
+          fi
+
+          echo ::set-output name=key_vault_name::$KEY_VAULT_NAME
+          echo ::set-output name=paas_space::$PAAS_SPACE
+        shell: bash
+        env:
+          TFVARS: workspace_variables/review.tfvars.json
+        working-directory: terraform
+
+      - uses: Azure/login@v1
+        with:
+          creds: ${{ secrets.azure_credentials }}
+
+      - uses: Azure/get-keyvault-secrets@v1
+        id: get_secrets
+        with:
+          keyvault: ${{ steps.config.outputs.key_vault_name }}
+          secrets: "TFSTATE-CONTAINER-ACCESS-KEY,PAAS-USER,PAAS-PASSWORD"
+
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.0.10
+          terraform_wrapper: false
+
+      - name: Set Environment variables
+        run: |
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          TF_STATE_FILE=review-pr-$PR_NUMBER.tfstate
+          echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
+          echo "TF_STATE_FILE=$TF_STATE_FILE" >> $GITHUB_ENV
+          pr_state_file=$(az storage blob list -c afqts-tfstate \
+            --aacount-key ${{ steps.get_secrets.outputs.TFSTATE-CONTAINER-ACCESS-KEY }} \
+            --prefix $TF_STATE_FILE --query "[].name" -o tsv)
+          if [ -n "$pr_state_file" ]; then echo "TF_STATE_EXISTS=true" >> $GITHUB_ENV; fi;
+
+      - uses: DFE-Digital/github-actions/setup-cf-cli@master
```

Comment:
This could also be removed from https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/.github/workflows/actions/deploy-environment/action.yml#L61

Reply:
no we don't. Will remove.